pnpm-lock.yaml

```diff
@@ -4733,7 +4745,7 @@ packages:
     dev: true
 
   /@pnpm/error/2.0.0:
-    resolution: {integrity: sha512-mgj4h0LWGpDPZwsEH75VFQhr2Njut3PcaCQatERIoO3zmKGqCsLfla9cWYH9+zn0fcwnKhnJ+FBzoiY2LhnCtw==}
+    resolution: {integrity: sha1-LKFAo70msHBAXo7scOEh5loUdeA=, tarball: '@pnpm/error/-/@pnpm/error-2.0.0.tgz'}
```

Comment:
this looks like not correct path to the tarball. What registry do you use locally?

Reply:
I have updated the lock file, sorry i just use the company registry...